CMakeLists.txt

```diff
@@ -77,6 +77,7 @@ cmake_dependent_option(BUILD_SAMPLES "Build samples?" ON "BUILD_DISPATCHER" OFF
 # Tools depend on samples (sample_common) and can't be built without it. The
 # following BUILD_TOOLS option declaration assures that.
 cmake_dependent_option(BUILD_TOOLS "Build tools?" "${BUILD_ALL}" "BUILD_SAMPLES" OFF)
+option(BUILD_TESTS "Build tests?" "${BUILD_ALL}")
```

Comment:
"Build unit tests?"

Reply:
this can be not only unit tests going forward. I think generic "Build tests?" is enough for now.

Reply:
ok, but now it will mean build and run unit tests. Probably "build tests" should just build all tests.

Reply:
Building and running of the tests is decoupled from each other. We don't have any other as of now.